From 347636b63e951894e35f3d3f0a66bf8684c55b61 Mon Sep 17 00:00:00 2001
From: Guy Morand <guy.morand@bytesatwork.ch>
Date: Wed, 3 Jul 2024 14:45:49 +0200
Subject: [PATCH] plat-stm32mp1: conf: Add RAM flavor for bytedevkit

This forces using 512Mb of RAM and fix a compilation issue where the
automatic configuration didn't match the content of the device tree.

Signed-off-by: Guy Morand <guy.morand@bytesatwork.ch>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index ef6fc368e..856176230 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -13,12 +13,16 @@ flavor_dts_file-157D_EV1 = stm32mp157d-ev1.dts
 flavor_dts_file-157F_DK2 = stm32mp157f-dk2.dts
 flavor_dts_file-157F_ED1 = stm32mp157f-ed1.dts
 flavor_dts_file-157F_EV1 = stm32mp157f-ev1.dts
+flavor_dts_file-157C_BDK-1_1 = stm32mp157c-bytedevkit-v1-1.dts
+flavor_dts_file-157C_BDK-1_3 = stm32mp157c-bytedevkit-v1-3.dts
 
 flavor_dts_file-135F_DK = stm32mp135f-dk.dts
 
 flavorlist-cryp-512M = $(flavor_dts_file-157C_DK2) \
 		       $(flavor_dts_file-157F_DK2) \
-		       $(flavor_dts_file-135F_DK)
+		       $(flavor_dts_file-135F_DK) \
+		       $(flavor_dts_file-157C_BDK-1_1) \
+		       $(flavor_dts_file-157C_BDK-1_3)
 
 flavorlist-no_cryp-512M = $(flavor_dts_file-157A_DK1) \
 			  $(flavor_dts_file-157D_DK1)
-- 
2.39.2

